optimum/exporters/openvino/__main__.py

```diff
@@ -328,6 +347,12 @@ class StoreAttr(object):
         if generation_config is not None:
             generation_config.save_pretrained(output)
         maybe_save_preprocessors(model_name_or_path, output)
+        try:
+            # TODO: Avoid loading the tokenizer again if loaded before
+            tokenizer = AutoTokenizer.from_pretrained(model_name_or_path)
+            tokenizer_export(tokenizer, output)
+        except:
+            print("[ WARNING ] Could not load tokenizer using specified model ID or path. OpenVINO tokenizer/detokenizer models won't be generated.")
```

Comment:
tokenizer already loaded in maybe_load_preprocessors function, I recommend to check this function result instead. Also possibly you should take into account trust_remote_code parameter if you want load tokenizer explicitly

Reply:
Reuse tokenizer from `maybe_load_preprocessors` result. But a new problem is that there are two identical tokenizers there - one from `AutoTokenizer` and one from `AutoProcessor`.

Reply:
Here is the PR: https://github.com/slyalin/optimum-intel/pull/2